### PR TITLE
Add documentation for OTP sensor

### PR DESCRIPTION
**Pull request by postlund**
**Description:**
Documentation for the OTP sensor.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#8332

### CODE REVIEW

File: source/_components/sensor.otp.markdown [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+```
+
+Configuration variables:
+
+- **name** (*Optional*): Name of the sensor to use in the frontend. Defaults to `OTP Sensor`.
+- **token** (*Required*): The shared secret you use in your OTP generator (e.g. Google Authenticator on your phone)
+
+## Generating a token
+
+A simple way to generate a ``token`` for a new sensor is to run this snippet of python code in your Home Assistant virtual environment:
```
Comment (Landrash) [inline source/_components/sensor.otp.markdown]: Only need single quotes around `token`

File: source/_components/sensor.otp.markdown [hunk 2]
```diff
@@ -0,0 +1,46 @@
+---
+layout: page
+title: "OTP Sensor"
+description: "Instructions how to add One-Time Passwod (OTP) sensors into Home Assistant."
```
Comment (Landrash) [inline source/_components/sensor.otp.markdown]: Misspelled word. 
Passwod->Password

File: source/_components/sensor.otp.markdown [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+Configuration variables:
+
+- **name** (*Optional*): Name of the sensor to use in the frontend. Defaults to `OTP Sensor`.
+- **token** (*Required*): The shared secret you use in your OTP generator (e.g. Google Authenticator on your phone)
+
+## Generating a token
+
+A simple way to generate a ``token`` for a new sensor is to run this snippet of python code in your Home Assistant virtual environment:
+
+```
```
Comment (Landrash) [inline source/_components/sensor.otp.markdown]: Add shell to highlight accordingly.
```shell
$ pip install pyotp
$ python -c 'import pyotp; print("Token: " + pyotp.random_base32())'
Token: IHEDPEBEVA2WVHB7
```

**Comment by postlund** [comment]
Thanks @Landrash! I've fixed your comments now! :+1:

**Pull request merged**